packages/app-project/pages/projects/[owner]/[project]/about/index.js

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
-
-const AboutProject = () => {
-  return (
-    <div>
-      This is the About Project landing page
-    </div>
-  )
-}
+import getDefaultPageProps from '@helpers/getDefaultPageProps'
+export { default } from '@screens/ProjectAboutPage'
+
+export async function getServerSideProps({ params, query, req, res }) {
+  const { props } = await getDefaultPageProps({ params, query, req, res })
+  const { env } = query
+  const { project } = props.initialState
+
+  const projectPath = `/projects/${project?.slug}/about/research`
```

Comment:
Non-blocking comment: If the project is undefined or null, should we attempt to route to this page? Perhaps routing to a 404 would be better, but ok to save for a future enhancement since I think 404 pages are an overall enhancement that needs work.